docs/demo.md

```diff
@@ -0,0 +1,227 @@
+- quarkus
+
+1. Open VSCode and load coolstore project if it is not already loaded.
+2. Follow steps from 1 through 4 from https://github.com/konveyor-ecosystem/kai/tree/main/ide#usage-summary
+3. Once the analysis is complete you will see incidents listed on the following files
+
+- `src/main/java/com/redhat/coolstore/model/ShoppingCart.java`
+- `src/main/java/com/redhat/coolstore/model/InventoryEntity.java`
+- `src/main/java/com/redhat/coolstore/service/CatalogService.java`
+- `src/main/java/com/redhat/coolstore/service/ShippingService.java`
+- `src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java`
+
+The incidents in the above files will provide insights into potential issues or areas that require attention during the migration process.
+
+#### 2.2.1 Change import namespaces
+
+- Right-click on the file `src/main/java/com/redhat/coolstore/model/ShoppingCart.java`.
+- Select Kai Fix-All.
+- Accept the proposed changes.
+- Repeat the same process for the file - `src/main/java/com/redhat/coolstore/model/InventoryEntity.java`.
+
+![Change import namespaces](model.png)
+
+Th above steps show how Kai simplifies the translation of import namespaces, ensuring seamless automated migration of javax libraries to jakarta persistence libraries.
+
+#### 2.2.2 Modify Scope from CDI bean requirements
+
+In this step, we will use Kai to modify the scope in `CatalogService.java` to adhere to Quarkus CDI bean requirements. Kai will handle this automatically, ensuring seamless migration.
+
+- Right-click on the file `src/main/java/com/redhat/coolstore/service/CatalogService.java`.
+- Select Kai Fix-All.
+- Accept the proposed changes.
+
+![Modify Scope from CDI bean requirements](kai_type2.png)
+
+#### 2.2.3 EJB Remote and Message Driven Bean(MDB) changes
+
+We will address EJB Remote and MDB functionalities in `ShippingService.java` and `ShoppingCartOrderProcessor.java` respectively. Kai will guide us through replacing EJBs with REST functionality and updating related imports and annotations.
+
+##### EJB Remote
+
+- Right-click on the file `src/main/java/com/redhat/coolstore/service/ShippingService.java`.
+- Select `Kai Fix-All`.
+- Accept the proposed changes.
+
+After applying the changes, you may notice unwanted imports such as `import jakarta.ejb.Singleton;`. Remove this import and save the file.
+
+![EJB Remote - Before/After](ejb_remote.png)
+
+Due to the absence of support for Remote EJBs in Quarkus, you will notice that these functionalities are removed and replaced with REST functionality.
+
+##### Message Driven Bean(MDB)
+
+- Right-click on the file `src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java`.
+
+- Select `Kai Fix-All` and accept the changes.
+
+- You will notice unwanted imports like `import jakarta.ejb.Singleton;`. Remove this import from the file and save it.
+
+- Check for any missing imports. If you find `import jakarta.enterprise.context.ApplicationScoped;` missing, add it to the file.
```

Comment:
Using cached data I see this import was added
<img width="1718" alt="Screenshot 2024-05-21 at 1 48 24 PM" src="https://github.com/konveyor-ecosystem/kai/assets/206875/d5ff6729-65c3-4f3c-aa3f-ea937080ae4f">